src/TerminalFighter/TargetingSystem/TargetingSystem.cpp

```diff
@@ -102,8 +102,20 @@ void TargetingSystem::update() {
 void TargetingSystem::draw(I_GraphicsHandler& graphics) {
     for (std::map<int, GameObjectStringPair*>::iterator it = targets_.begin(); it != targets_.end(); ++it) {
         SDL_Surface* UIText = TTF_RenderText_Blended(default_font_, it->second->assigned_word_.c_str(), TF_Colors::WHITE);
-        graphics.draw(UIText, (int)it->second->game_object_.x_pos(), (int)it->second->game_object_.y_pos(),
-                      GraphicPriority::UI, true);
+        SDL_Rect ui_src_rect;
+        ui_src_rect.x = 0;
+        ui_src_rect.y = 0;
+        ui_src_rect.w = UIText->w;
+        ui_src_rect.h = UIText->h;
```

Comment:
The utility func should be used here. Also why is UIText cap case? Shouldn't it be snake_case?

Reply:
Ok, I'll have to create another utility function, probably overload it. But the original utility function took SDL_Texture as a argument, that's why I didn't use it here.